Makefile

```diff
@@ -89,11 +94,17 @@ forex-plugins:
 	go build -o $(PLUGIN_DIR)/forex_openexchange $(PLUGIN_SRC_DIR)/forex_openexchange/forex_openexchange.go
 	chmod +x $(PLUGIN_DIR)/*
 
+usdc-plugins:
+	go build -o $(PLUGIN_DIR)/usdc_coinbase $(PLUGIN_SRC_DIR)/usdc_coinbase/usdc_coinbase.go
+	go build -o $(PLUGIN_DIR)/usdc_coingecko $(PLUGIN_SRC_DIR)/usdc_coingecko/usdc_coingecko.go
+	go build -o $(PLUGIN_DIR)/usdc_kraken $(PLUGIN_SRC_DIR)/usdc_kraken/usdc_kraken.go
+	chmod +x $(PLUGIN_DIR)/*
+
 piccadilly-cax-plugin:
 	go build -o $(PLUGIN_DIR)/pcgc_cax $(PLUGIN_SRC_DIR)/pcgc_cax/
 	chmod +x $(PLUGIN_DIR)/pcgc_cax
 	# cp autonity round4 game PCGC CAX plugins for e2e testing
-	cp $(PLUGIN_DIR)/pcgc_cax $(E2E_TEST_CAX_PLUGIN_DIR)/pcgc_cax
+	cp $(PLUGIN_DIR)/pcgc_cax $(E2E_TEST_CRYPTO_PLUGIN_DIR)/pcgc_cax
```

Comment:
The CAX was for the Piccadilly Circus Games, so I think this plugin can be removed as legacy.

Reply:
A plugin to get prices from on-chain AMM will be required.

Reply:
As we haven't implement the extra plugin of the onchain AMM data source, let's keep it.

Reply:
OK